fyle_rest_auth/helpers.py

```diff
@@ -44,11 +44,63 @@ def validate_code_and_login(request):
         tokens['user']['org_name'] = employee_info['data']['org']['name']
 
         # Update Fyle Credentials with latest healthy token
-        async_task(
-            'apps.workspaces.tasks.async_update_fyle_credentials',
-            employee_info['data']['org']['id'], tokens['refresh_token']
+        if 'async_update_user' in settings.FYLE_REST_AUTH_SETTINGS \
+             and settings.FYLE_REST_AUTH_SETTINGS['async_update_user']:
```

Comment:
I checked the file was not able to find the import for the `settings` . Also i have pulled the latest master on local not able to locate the settings file.

Reply:
Import is there at line 6